pkg/server/api_v2_databases_metadata.go

```diff
@@ -888,3 +1006,8 @@ type tmJobTriggeredResponse struct {
 	JobTriggered bool             `json:"job_triggered"`
 	Message      JobStatusMessage `json:"message"`
 }
+
+type tableMetadataForIdResponse struct {
```

Comment:
nit: Can we rename this to something like `tableMetadataWithDetails` to signal it's the table metadata with some more information?

Reply:
Done